rdkit/Chem/Draw/__init__.py

```diff
@@ -573,6 +571,238 @@ def _MolsToGridImage(mols, molsPerRow=3, subImgSize=(200, 200), legends=None,
   return res
 
 
+def _MolsNestedToLinear(mols_matrix, legends_matrix = None, highlightAtomLists_matrix = None,
```

Comment:
General comment on naming: the RDKit generally doesn't use underscores to separate words in variable names (there are instances that slipped through, but they are the minority). Please try to make the code style consistent with the rest of the RDKit.